App/iOS/Delegates/SceneDelegate.swift

```diff
@@ -216,6 +216,10 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
   }
 
   func sceneWillResignActive(_ scene: UIScene) {
+    if let scene = scene as? UIWindowScene, let browserViewController = scene.browserViewController {
```

Comment:
You could technically do this by using `NotificationCenter` and listening for the `resignActive` which is already done inside `BrowserViewController.swift` :D 

https://github.com/brave/brave-ios/blob/development/Sources/Brave/Frontend/Browser/BrowserViewController.swift#L695

but approved anyway, as this works too. If you want to change it, feel free to do so.

Reply:
Changed at https://github.com/brave/brave-ios/pull/7551/commits/1fc4136f25a1bdf21ee0abf11db2de63c5dc4ae5